src/views/officer.j2

```diff
@@ -1,9 +1,10 @@
 <p><b>Badge #:</b> {{ Serial }}</p>
 <p><b>Name:</b> {{ name }}</p>
 <p><b>Rank:</b> {{ RankRole }}</p>
-<p><b>Unit Description:</b> {{ UnitDesc }}</p>
+<p><b>Unit Description*:</b> {{ UnitDesc }}</p>
```

Comment:
I wonder if it might be better to just note this at the top instead of for every line? Of course having it in every line is good for screenshots.

Reply:
I've seen a bunch of screenshots so far where this makes it in, so I think it might be best to have it for each one. Good thought though!